scripts/gh-summary.py

```
@@ -414,6 +416,47 @@ def render_report(
+    """Fetch recently updated open issues from a GitHub repository.
+
+    Args:
+        repo: ``owner/name`` string for the target repository.
+        limit: Maximum number of issues to fetch. Defaults to 10.
+
+    Returns:
+        The JSON array returned by ``gh issue list`` as a list of issue
+        dictionaries, or an empty list when ``gh`` fails or returns
+        invalid JSON.
+    """
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)